backend/app/models/llm/request.py

```
@@ -223,3 +232,210 @@ class LLMCallRequest(SQLModel):
+
+class LlmCall(SQLModel, table=True):
+    """
+    Database model for tracking LLM API call requests and responses.
+
+    Stores both request inputs and response outputs for traceability,
+    supporting multimodal inputs (text, audio, image) and various completion types.
+    """
+
+    __tablename__ = "llm_call"
+    __table_args__ = (
+        Index(
```

Ruff (0.14.14)

[error] Undefined name `Index`

(F821)